packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -141,13 +141,14 @@ export class PluginTree extends TreeImpl {
             }, update);
         }
         if (TreeViewNode.is(node)) {
-            return Object.assign(node, update);
+            return Object.assign(node, update, { command: item.command });
```

Comment:
@vinokurig command cannot be applied to composite elements? just checking

Reply:
yes, according to previous logic that was lost:
https://github.com/theia-ide/theia/blob/93205109fa4017286ec0a441b0250b74b36c6495/packages/plugin-ext/src/main/browser/view/tree-views-main.tsx#L206-L218
command is set to `fileNode`, bit is not set to `folderNode`:
https://github.com/theia-ide/theia/blob/93205109fa4017286ec0a441b0250b74b36c6495/packages/plugin-ext/src/main/browser/view/tree-views-main.tsx#L189-L204

Reply:
I meant whether it should be set on the composite according to VS Code expectations? If so it would be nice to adjust it as well. It could be done separately, but needs an issue at least.

Reply:
Raised an issue with question https://github.com/theia-ide/theia/issues/6025